docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -305,6 +305,7 @@ Valid options for `[Container]` are listed below:
 | AddDevice=/dev/foo                   | --device /dev/foo                                    |
 | AddHost=example\.com:192.168.10.11   | --add-host example.com:192.168.10.11                 |
 | Annotation="XYZ"                     | --annotation "XYZ"                                   |
+| AppArmor="alternate-profile" | --security-opt apparmor=alternate-profile |
```

Comment:
To keep my OCD in check, can you line up the `|` with the lines above/below please?

Reply:
Perfectly valid, I fixed this. I had to turn off my Markdown formatting because it got me in trouble on a different PR :)